types/wkhtmltopdf/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "esModuleInterop": true,
```

Comment:
```suggestion
```
can you remove this? It's for TS consumption (to ease devex), not useful here in DT contexts, and this package is a plain, CJS module